### PR TITLE
Create pptoggle.java

### PR DESCRIPTION
**Pull request by Telesphoreo**
don't know what to do when the plugin isn't there

also first time trying to use github

### CODE REVIEW

**Comment by PacksGamingHD** [comment]
#831

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    				playerparticles.getPluginLoader().disablePlugin(playerparticles);
+    				enabled = false;
+    			} else {
+    				playerparticles.getPluginLoader().enablePlugin(playerparticles);
+    				enabled = true;
+    			}
+    			toggled = true;
+    		}
+    	}
+    	if (toggled) {
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: Merge this If statement in the previous for loop please.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {	
+    	boolean toggled = false;
+    	boolean enabled = true;
+        
+    	for (Plugin playerparticles : TotalFreedomMod.server.getPluginManager().getPlugins()) {
+    		if (playerparticles.getName().equalsIgnoreCase("playerparticles")) {
+    			if (playerparticles.isEnabled()) {
+    				playerparticles.getPluginLoader().disablePlugin(playerparticles);
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: Incorrect indentation.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import org.bukkit.plugin.Plugin;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Turns on or off the PlayerParticles plugin", usage = "/<command>")
+public class Command_pptoggle extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {	
+    	boolean toggled = false;
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: Don't need these booleans.

**Comment by PacksGamingHD** [comment]
I had it but I lost all my work for some reason -sigh-

**Comment by PacksGamingHD** [comment]
well the indentation messed up from what my netbeans was...

**Comment by LifeAsStranger** [comment]
This code format is needed to be cleaned up.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+@CommandParameters(description = "Turns on or off the PlayerParticles plugin", usage = "/<command>")
+public class Command_pptoggle extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {	
+        
+    	for (Plugin p : TotalFreedomMod.server.getPluginManager().getPlugins()) {
+    		if (p.getName().equalsIgnoreCase("playerparticles")) {
+    			if (p.isEnabled()) {
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: Formatting

**Comment by OxLemonxO** [comment]
instead of else { it should be
else 
{

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Turns on or off the PlayerParticles plugin", usage = "/<command>")
+public class Command_pptoggle extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {	
+    	for (Plugin p : TotalFreedomMod.server.getPluginManager().getPlugins()) {
+    	    if (p.getName().equalsIgnoreCase("PlayerParticles")) {
+    		    if (p.isEnabled()) {
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: Formatting.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+                    p.getPluginLoader().disablePlugin(p);
+    	            TFM_Util.adminAction(sender.getName(), "Disabling PlayerParticles", true);
+    	        } else {
+    		  p.getPluginLoader().enablePlugin(p);
+    		  TFM_Util.adminAction(sender.getName(), "Enabled PlayerParticles", false);
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: Inconsistent message.

Comment (PacksGamingHD) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: What does that mean @JeromSar

**Comment by PacksGamingHD** [comment]
Sorry @OxLemonxO I'm used to doing it that way

**Comment by JeromSar** [comment]
Formatting, formatting and more formatting.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    for (Plugin p : TotalFreedomMod.server.getPluginManager().getPlugins()) {
+    if (p.getName().equalsIgnoreCase("PlayerParticles")) {
+    if (p.isEnabled()) {
+        p.getPluginLoader().disablePlugin(p);
+        TFM_Util.adminAction(sender.getName(), "Disabling PlayerParticles", true);
+        } 
+        else 
+        {
+            p.getPluginLoader().enablePlugin(p);
+            TFM_Util.adminAction(sender.getName(), "Enabled PlayerParticles", false);
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: Inconsistent color

**Comment by OxLemonxO** [comment]
at the end, true means red.
both the colors should be the same.
Also, consistent messages please:
Enabling:
Enabled PlayerParticles
Disabling:
Disbled PlayerPaticles

**Comment by OxLemonxO** [comment]
and instead of this:
for {
if {
else {
or anything like that
use
for 
{
if
{
else
{
etc.
Same with the }'s.

**Comment by PacksGamingHD** [comment]
Lemon, I know true means red. I intended it to be red when disabling and blue when enabling. The format is correct on my end but not for everyone else. I'll do the consistent messages now

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+        }
+            }
+        }
+return true;
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: Formatting.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+import org.bukkit.plugin.Plugin;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Turns on or off the PlayerParticles plugin", usage = "/<command>")
+public class Command_pptoggle extends TFM_Command
+{
+@Override
+public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+{	
+    for (Plugin p : TotalFreedomMod.server.getPluginManager().getPlugins()) {
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_pptoggle.java]: Formatting above and below.

**Pull request closed without merging**